### PR TITLE
Update Program.cs

### PR DESCRIPTION
**Pull request by M2203114**
I hope now it is clear

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -4,167 +4,66 @@ namespace _1st_Lesson
...
-            if (Int32.TryParse(str, out int numberFromString))
+            Task3();
+        }
+        public static void Task1()
+        {
+            Console.WriteLine("Height in santimeters: ");
+            double height = Convert.ToDouble(Console.ReadLine())/100;//182
+            Console.WriteLine("Weight in kilograms: ");
+            double weight = Convert.ToDouble(Console.ReadLine());//78
+            if (height == 0)
```
Comment (M2203114) [inline Program.cs:17]: (height<1)

File: Program.cs [hunk 2]
```diff
@@ -4,167 +4,66 @@ namespace _1st_Lesson
...
+        public static void Task1()
+        {
+            Console.WriteLine("Height in santimeters: ");
+            double height = Convert.ToDouble(Console.ReadLine())/100;//182
+            Console.WriteLine("Weight in kilograms: ");
+            double weight = Convert.ToDouble(Console.ReadLine());//78
+            if (height == 0) 
             {
-                divizionByZero /= numberFromString;
+                Console.WriteLine("height can't be 0, enter a real one");
```
Comment (M2203114) [inline Program.cs:19]: cant be 0 or lower

File: Program.cs (hunk 1 above)
Comment (M2203114) [reply Program.cs:17]: also weight, but I dont know how to edit file

**Comment by NemirovichMark** [comment]
Hello.

1. Add name for your profile, please.

2. You can edit your file like this:
image
image
image

3. Remove everything that weren't asked in the tasks.


4. Make a request to your branch in my repository instead master branch. Look at the presentation "How to work with GitHub" on your mail.

5. You did the verification from wrong value, but than get the same error. At lines 62-64.
If the input was invalid, message about mistake is enough. I don't claim to get right answer at all cost.

You have 2 attempts left.

**Pull request closed without merging**